<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SceneInformations_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene informations</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>11</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select from list</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="objectName">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFbxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open FBX</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="All_tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="entitiesSi_all">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>33</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pointsSi_all">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pointsSi_all_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Points : </string>
      </property>
     </widget>
     <widget class="QLabel" name="polygonsSi_all_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Polygons :</string>
      </property>
     </widget>
     <widget class="QLabel" name="polygonsSi_all">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="entitiesSi_all_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Entities</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Selected_tab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="entitiesSi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Entities</string>
      </property>
     </widget>
     <widget class="QLabel" name="entitiesSi_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pointsSi">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pointsSi_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Points : </string>
      </property>
     </widget>
     <widget class="QLabel" name="polygonsSi_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Polygons :</string>
      </property>
     </widget>
     <widget class="QLabel" name="polygonsSi">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="transformFrame">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rotation_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QLabel" name="zSca_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="xRot_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="xSca_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="position_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="translateBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
    <widget class="QLabel" name="yPos_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yRot">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zSca">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="xRot">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ySca">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QLabel" name="zPos_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scaleBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QLabel" name="ySca_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yPos">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="zRot_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="scale_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zRot">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="xSca">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zPos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="xPos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="xPos_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="yRot_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="objectName_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openShaderEditor_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Shader Editor</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileUrl_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>741</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="topView" rowstretch="0" columnstretch="0" rowminimumheight="0"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
